### ISSUE TITLE
Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift

### ISSUE THREAD

**Issue by sooooner** [sooooner/papers#18]
# Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift[[link](https://arxiv.org/abs/1502.03167)]

## Abstract
network학습시 layer의 input은 계속 변화한다
이는 가중치 초기화, 낮은 학습률을 야기하기 때문에 학습속도가 늦어진다. 또한saturating nonlinearity도 유발한다.
이 현상을 Internal Covariate Shift라고 하며 해결방법을 제시한다.

## Introduction
기존 network학습에는 가중치 초기화, 학습률 선택등등을 고려해야 한다.
또한 saturating nonlinearity를 해결하기 위해 activation을 바꾸었다. 

이를 layer관점에서 Batch Normalization을 통해 해소해보겟다.

## Towards Reducing Internal Covariate Shift
훈련데이터가 정규분포를 따르면 좋다는건 알려져있다.
그래서 activation을 whitning하는방식을 생각해봤지만 backprop이 이상하게 작동한다.
특정 layer의 output을 단순히 whitning하는 방식 또한 optimization과정에서 문제가 발생힌다.

즉 위의 문제를 일으키지 않으면서 layer output의 분포가 정규화되는 방식을 찾아야 하낟.

## Normalization via Mini-Batch Statistics
whitening말고  대신에 feature에 대하여 normalize를 독립적으로 수행했다. (전체 평균,분산으로)
문제점! sigmoid는 [-1,1]에서 거의 선형임!

따라서 두개의 학습가능한 추가 변수 beta와 gamma를 사용해 ![image](https://user-images.githubusercontent.com/31437443/108807371-943f3e80-75e7-11eb-8fcd-be6e769d8bf0.png)로 조정한 값을 최종 output으로 사용한다.

![image](https://user-images.githubusercontent.com/31437443/108807345-838ec880-75e7-11eb-8763-6c529b616fee.png)
